ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -17,13 +17,14 @@ http {
     access_log /logs/nginx_access.log combined-upstream;
 
     server {
+        listen 80;
```

Comment:
Fair - the http here is directly communication with the API controller not the edge router. So some features will not work (ie cannot download the docker or iOS SDK for example).

FYI @csantanapr (and @akrabat).